app/main.py

```diff
@@ -1,6 +1,15 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args) -> Any:
+        if args in cache_dict:
+            print("Getting from cache")
```

Comment:
as key for cache_dict usetuple with *args, **kwargs and name of function since all of this will define unique call of function.